app/build.gradle

```diff
@@ -74,6 +77,18 @@ android {
         kotlinCompilerExtensionVersion '1.3.0'
     }
     namespace 'org.listenbrainz.android'
+    compileOptions {
```

Comment:
We don't need these additions. Please remove

Reply:
Done